src/Utils/InitialConditions/init_3d_rising_bubble.jl

```diff
@@ -18,7 +20,13 @@
 
     # TODO!: Replace expression with Thermodynamics.jl expressions.
 """
-function init_3d_rising_bubble(::Type{FT}, params, thermovar = :ρθ) where {FT}
+
+function init_3d_rising_bubble(
+    ::Type{FT},
+    params;
+    thermodynamics,
+    moist,
```

Comment:
lets call it `thermo_mode` or `thermo_style` and `moisture_style` so that it is super unambiguous.